### DISCUSSION TITLE
how to compile Files or Directory when wails build -nsis?

### DISCUSSION THREAD

**Discussion by askie** [wailsapp/wails#1540] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

I have some files must be pulish with wails build exe file,
when I use wails build -nsis , how to config ?

### Describe the solution you'd like

need a solution

**Comment by stffabi** [comment, marked as answer]
After using the `wails build -nsis` you will find your `project.nsi` file in the folder `build/windows/installer`. The nsi file is a standard NSIS installer script file. Please refer to the [nsis documentation](https://nsis.sourceforge.io/Docs/) for a detailed description on how to write or enhance NSIS script files.
